src/vs/workbench/browser/parts/editor/editor.ts

```diff
@@ -340,9 +340,9 @@ export interface IInternalEditorOpenOptions extends IInternalEditorTitleControlO
 	readonly preserveWindowOrder?: boolean;
 
 	/**
-	 * Whether to add the editor to the selection or not.
+	 * Inactive editors to select after opening the active selected editor.
 	 */
-	readonly selected?: boolean;
+	readonly inactiveSelectedEditors?: EditorInput[];
```

Comment:
I think we are getting closer: my feeling is we can remove this option entirely and change the only usage to this:

```ts
async setSelection(activeSelectedEditor: EditorInput, inactiveSelectedEditors: EditorInput[]): Promise<void> {
	if (!this.model.isActive(activeSelectedEditor)) {
		await this.doOpenEditor(activeSelectedEditor, { activation: EditorActivation.ACTIVATE });
	}
	this.model.setSelection(activeSelectedEditor, inactiveSelectedEditors);
}
```